compose/docker-compose.yaml

```diff
@@ -33,6 +33,30 @@ services:
       - 80:80
     networks:
       - flam-flam
+    depends_on:
+      - mongodb
+
+  mongodb:
+    image: mongo:6.0
+    restart: always
+    pull_policy: missing
+    environment:
+      MONGO_INITDB_DATABASE: flam
+      MONGO_INITDB_ROOT_USERNAME: ${MONGO_INITDB_ROOT_USERNAME}
+      MONGO_INITDB_ROOT_PASSWORD: ${MONGO_INITDB_ROOT_PASSWORD}
+    volumes:
+      - ./mongodb/comments.js:/docker-entrypoint-initdb.d/comments.js
+      - ./mongodb/submissions.js:/docker-entrypoint-initdb.d/submissions.js
```

Comment:
It all works, but do these .js files live anywhere? Docker creates them as directories in my case:

```
docker logs f22610fcd7dc | grep comm
ents
Error: EISDIR: illegal operation on a directory, read
/usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/comments.js
```

From what I understand, we could use those scripts to [init the database](https://faun.pub/initialize-mongodb-running-on-a-docker-container-889a43c5668a) with some collections OR just do that within the services themselves? In the latter case we can just create a user for services to connect to. Not sure how to deal with secrets there (well, it's easy in Helm :D ).

Reply:
Damn it, I forgot to take those lines out - I put them in when I thought I needed to specifically create the databases and now I've just remembered that this is where I got stuck because it didn't work how I thought it would 🤦🏻